Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
```

Comment:
What is weird is that i'm not having that problem here.  I can run `Texture2DTest` for instance and it seems to be working.

How are you running the tests?  Maybe we are doing it differently?